src/main/java/org/cloudfoundry/community/servicebroker/s3/config/ServiceConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package org.cloudfoundry.community.servicebroker.s3.config;
+
+import org.cloudfoundry.community.servicebroker.model.Plan;
+import org.cloudfoundry.community.servicebroker.model.ServiceDefinition;
+import org.cloudfoundry.community.servicebroker.s3.plan.basic.BasicPlan;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Created by jcarter on 22/01/16.
+ */
+@Configuration
+public class ServiceConfiguration {
+
+    @Value("${SERVICE_ID:s3}")
```

Comment:
this is spring expression it should be `SERVICE.ID` and not `SERVICE_ID` (spring will take the env var `SERVICE_ID` and make it available as `SERVICE.ID` in java) 

It needs the same for `SERVICE_NAME`, `PLAN_ID` and `PLAN_NAME`